src/styles/shared/_controls.scss

```diff
@@ -11,7 +11,48 @@
 }
 
 @function control-icon-transition() {
-  @return transform duration(fast) easing(in) easing(out);
+  @return transform duration(fast) easing(in);
+}
+
+@mixin control-backdrop-dl($style: base) {
+  @if $style == base {
```

Comment:
created a new mixin again to keep things simple.

Reply:
As I indicated before, might it make more sense to stick with the same mixin, but add a property for turning on the new design language?